src/quo2/components/inputs/profile_input/view.cljs

```diff
@@ -0,0 +1,52 @@
+(ns quo2.components.inputs.profile-input.view
+  (:require
+    [quo2.components.buttons.button :as buttons]
+    [quo2.components.avatars.user-avatar.view :as user-avatar]
+    [quo2.components.inputs.profile-input.style :as style]
+    [quo2.components.inputs.title-input.view :as title-input]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [react-native.hole-view :as hole-view]))
+
+(defn profile-input
+  [{:keys [customization-color
+           placeholder
+           on-press
+           title-input-props
+           image-picker-props]}]
+  (let [full-name (:full-name image-picker-props)]
+    [rn/view
+     {:style (style/container customization-color)}
+     [rn/view {:position :relative}
+      [hole-view/hole-view
+       {:style {}
+        :holes [{:x            33
+                 :y            23
+                 :width        24
+                 :height       24
+                 :borderRadius 12}]}
+       [user-avatar/user-avatar
+        (merge image-picker-props
+               {:customization-color customization-color
+                :full-name           (if (zero? (count full-name))
```

Comment:
Strings are also *sequences* in Clojure, so we don't need to check if the number of elements is zero. Also we have the more solid `clojure.string/blank?` that takes whitespace into account, but I don't know if it'd add much value here.

```clojure
(let [full-name ""]
  (if (seq full-name)
    full-name
    "I'm a placeholder")) ;; => I'm a placeholder

(let [full-name "someone"]
  (if (seq full-name)
    full-name
    "placeholder")) ;; => someone
```